.storybook/__fixtures__/_googlesitekitLegacyData.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * Legacy global data fixture.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default {
+	admin: {
+		siteURL: 'https://example.com',
+		siteName: 'My Great Blog',
+		siteUserID: 'a1b2c3d4e5f6a1b2c3d4e5f6a1b2c3d4',
+		adminRoot: 'https://example.com/wp-admin/admin.php',
+		assetsRoot: 'https://example.com/wp-content/plugins/google-site-kit/dist/assets/',
+		nojscache: false,
+		datacache: false,
```

Comment:
`datacache` was the key that held the huge string of JSON that made this so massive before. It isn't used anymore although the key is still set.